charts/utils/Util.ts

```diff
@@ -1007,3 +1007,22 @@ export const getAvailableSlugSync = (
 
     return slug
 }
+
+/**
+ * Use with caution - please note that this sort function only sorts on numeric data, and that sorts
+ * **in-place** and **not stable**.
+ * If you need a more general sort function that is stable and leaves the original array untouched,
+ * please use lodash's `sortBy` instead. This function is faster, though.
+ */
+export function sortNumeric<T>(
```

Comment:
Nit: even though it's correct, might be worth it to add a unit test in the off chance in a year someone accidentally changes a sign or adds a param that breaks other things..

Reply:
Yup, was actually wondering myself if that makes sense - will do!